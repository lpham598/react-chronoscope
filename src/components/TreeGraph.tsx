import * as React from "react";
import Tree from 'react-d3-tree';

  
const myTreeData = [
  {
    name: 'App',
    attributes: {
      keyApp: 'val A',
    },
    children: [
      {
        name: 'A1',
        attributes: {
          keyA: 'val A',
        },
      },
      {
        name: 'A2',
      },
    ],
  },
];

  class TreeGraph extends React.Component {
    render() {
      return (
        // <div id="treeWrapper" style={{width: '50em', height: '20em'}}>
        <div id='treeGraph' style={{}}>

        <Tree data={myTreeData} orientation="vertical" translate={{ x: 100, y: 20}}/>
  
        </div>
      );
    }
  }

export default TreeGraph;