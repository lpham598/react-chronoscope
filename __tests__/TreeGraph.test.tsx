import React from 'react';
import { shallow } from 'enzyme';

import TreeGraph from '../src/components/TreeGraph';

describe('React-ChronoScope Component Tests', () => {
  describe('Component: TreeGraph', () => {
    let wrapper;

    beforeAll(() => {
      wrapper = shallow(<TreeGraph/>);
    });

    it('Renders a <div> tag', () => {
      expect(wrapper.type()).toEqual('div');
    });
  });
});