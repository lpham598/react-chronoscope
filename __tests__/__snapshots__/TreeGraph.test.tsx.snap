// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React-ChronoScope Component Tests Component: TreeGraph should render correctly with no props 1`] = `undefined`;
